### PR TITLE
added 3 entries and re-sorted

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/changes-device-management/cl-device-management.md [hunk 1]
```diff
@@ -38,6 +41,15 @@ In the Device management application, a wizard has been implemented which guides
 
 The Device management home page now also provides a customizable dashboard which lets users add customized widgets. [DM-1644]
 
+#### -Change Device management: Added information on restrictions to auto-registration
+
+Information has been added that the auto-registration option in trusted certification does not support devices communicating via the LWM2M protocol. This information has been included in the tooltip on the **Trusted certificates** page and in the user documentation. [MTM-56462]
```
Comment (rssag) [inline content/changes-device-management/cl-device-management.md:46]: either add the definite article or omit the phrase "information has been added (that)" - the second sentence basically repeats that.

**Review by rssag — APPROVED** [review]
one comment + approval

File: content/changes-device-management/cl-device-management.md (hunk 1 above)
Comment (BeateRixen) [inline content/changes-device-management/cl-device-management.md:46]:
```suggestion
The auto-registration option in trusted certification does not support devices communicating via the LWM2M protocol. This information has been included in the tooltip on the **Trusted certificates** page and in the user documentation. [MTM-56462]
```

File: content/changes-device-management/cl-device-management.md [hunk 2]
```diff
@@ -38,6 +41,15 @@ In the Device management application, a wizard has been implemented which guides
 
 The Device management home page now also provides a customizable dashboard which lets users add customized widgets. [DM-1644]
 
+#### -Change Device management: Added information on restrictions to auto-registration
```
Comment (Oezge-Akin) [inline content/changes-device-management/cl-device-management.md:44]:
```suggestion
#### -Change- Device management: Added information on restrictions to auto-registration
```

File: content/changes-device-management/cl-device-management.md [hunk 3]
```diff
@@ -38,6 +41,15 @@ In the Device management application, a wizard has been implemented which guides
...
+
+The auto-registration option in trusted certification does not support devices communicating via the LWM2M protocol. This information has been included in the tooltip on the **Trusted certificates** page and in the user documentation. [MTM-56462]
+
+#### -Feature LWM2M: New toggle for switching off empty data
```
Comment (Oezge-Akin) [inline content/changes-device-management/cl-device-management.md:48]:
```suggestion
#### -Feature- LWM2M: New toggle for switching off empty data
```

**Pull request merged**